pyvo/conftest.py

```diff
@@ -17,7 +17,7 @@
     ASTROPY_HEADER = False
 
 # Keep this until we require numpy to be >=2.0
-if minversion(np, "2.0.0.dev0+151"):
+if minversion(np, "2.0.0.dev0+git20230726"):
```

Comment:
I could probably use just the 2.0.0.dev0 tag, but that's technically not correct  and could cause failures for the unlikely scenario of someone having an a couple of weeks old dev version. So will just go with this.